packages/endpoint/endpoint.js

```diff
@@ -0,0 +1,154 @@
+import fs from 'fs';
+import axios from 'axios';
+import RestroomCore from "@restroom-mw/core";
+const { Restroom } = RestroomCore;
+
+const ACTIONS = {
+  EXTERNAL_CONNECTION: "that I have an endpoint named {}",
+  EXTERNAL_OUTPUT: "I connect to {} and save the output",
+  PASS_OUTPUT: "pass the output to {}"
+};
+
+export default (req, res, next) => {
+  const rr = new Restroom(req, res);
+  let externalSourceKeys = [];
+  let externalSourceValue = [];
+
+  let outputName = 'output';
+  let outputValue;
+
+  //get the api call from restroom that contains username and contract name
+  const originalUrl = rr._req.originalUrl;
+  const userContractDir = originalUrl.split('/api/')[1];
+
+  //Get the directories for keys and data
+  const completeDir = process.cwd() + '/contracts/' + userContractDir;
+  const keyFile = completeDir + '.keys';
+  const dataFile = completeDir + '.data';
```

Comment:
These paths are also specific for your software, should be generic to work also in other softwares. Probably you want to make this section is some way configurable. To access the folder where the contracts are you can use https://dyne.github.io/restroom-mw/#/packages/utils?id=zencode_dir like in https://github.com/dyne/restroom-mw/blob/master/packages/core/src/index.js#L2